src/drive/actions/index.jsx

```diff
@@ -249,8 +256,10 @@ export const uploadQueueProcessed = (loaded, quotas, conflicts, errors, t) => {
       smart_count: loaded.length,
       conflictNumber: conflicts.length
     })
+  } else if (networkErrors.length > 0) {
+    Alerter.info('upload.alert.network')
   } else if (errors.length > 0) {
-    Alerter.error('upload.alert.errors')
+    Alerter.info('upload.alert.errors')
```

Comment:
Why an `info` instead of `error`?

Reply:
`error` creates a bright red alert, which we should (as of now) only use when the user is responsible for the error. Here's it's more likely a script error somewhere, so we wanted a nicer looking message.